runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -113,8 +113,13 @@ public Dimension render(Graphics2D graphics)
 							return null;
 						}
 						// intentional fallthrough
-					case WidgetID.BANK_GROUP_ID:
 					case WidgetID.BANK_INVENTORY_GROUP_ID:
+						if (config.hideInventory())
```

Comment:
Just add this to the `WidgetID.INVENTORY_GROUP_ID` case above so the code is not duplicated.

Reply:
Move the case statment so the invetory check code isn't duplicated